src/Orchard.Web/Modules/Orchard.ImportExport/Services/ISetupServiceFactory.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Orchard.Setup.Services;
+
+namespace Orchard.ImportExport.Services {
+    /// <summary>
+    /// We need to manually instantiate the SetupService class because the Orchard.Setup feature will be disabled after setup completes.
+    /// </summary>
```

Comment:
Maybe this comment could include more context, such as:

    This factory service exists because ISetupService cannot be dependency injected due to the fact that the Orchard.Setup feature will be disabled after setup completes. Instead, this module takes only an assembly reference on Orchard.Setup and instantiates the SetupService implementation directly.